www/templates/splash.html

```diff
@@ -1,6 +1,8 @@
 <ion-view>
     <ion-content>
       <img id="splash-logo" src='img/logo.png'>
+      <p><a href="" ng-click="showAlert()"><center>Leia os termos de uso</center></a></p>
```

Comment:
pelo que me lembro, o elemento `center` caiu em desuso e também não possui suporte no HTML5. Acho que vale usar puro CSS seguindo o descrito aqui https://stackoverflow.com/questions/1798817/why-is-the-center-tag-deprecated-in-html